packages/opentelemetry-cloud-trace-exporter/package.json

```diff
@@ -37,11 +37,9 @@
     "registry": "https://wombat-dressing-room.appspot.com"
   },
   "devDependencies": {
-    "@opentelemetry/api": "0.7.0",
-    "@opentelemetry/base": "0.7.0",
-    "@opentelemetry/core": "0.7.0",
-    "@opentelemetry/resources": "0.7.0",
-    "@opentelemetry/tracing": "0.7.0",
+    "@opentelemetry/api": "0.8.0",
```

Comment:
The latest version is `0.10.2`

Reply:
Should it be kept at the oldest working version or the newest working version?

Reply:
If you ask me, I would  directly put new version. I think this is fine till we are in Beta version.